torch/quantization/fx/quantization_patterns.py

```diff
@@ -754,10 +754,8 @@ def convert(self,
                         bias = load_arg(quantized=torch.float)(self.linear_node.args[2])
                         other_args = other_args[1:]  # remove the bias argument
                     else:
-                        assert 'bias' in kwargs, \
-                            'expect bias provided as a keyword argument when it is not a positional argument'
```

Comment:
This also only works coincidentally. `bias` is an optional argument, but for python bound methods defaults are currently handled before `__torch_function__` but there's no guaruntee this is the case, and this isn't true for C++ bound functions.